### PR TITLE
open-cluster-management/backlog#2667 Networking

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: install/requirements.md [hunk 1]
```diff
@@ -24,3 +24,25 @@ You can access the Red Hat Advanced Cluster Management for Kubernetes console fr
...
 
 See the [Red Hat Advanced Cluster Management for Kubernetes Support matrix](https://access.redhat.com/articles/5058921) for additional information. 
+
+# Networking configuration
+
+Configure your network settings to allow the following connections:
+
+Hub cluster:
+- Outbound connectivity to cloud provider's API
+- Outbound connectivity to the Kubernetes API server of the Hive provisioned `ManagedCluster` on port 443 (so Hive-driven `openshift-install` is successful)
```
Comment (TheRealHaoLiu) [inline install/requirements.md]: port 6443

File: install/requirements.md [hunk 2]
```diff
@@ -24,3 +24,25 @@ You can access the Red Hat Advanced Cluster Management for Kubernetes console fr
...
+- Outbound connectivity from the Hub to the channel source, including Github, Object Store, and Helm repository
+- Outbound and inbound connectivity to the `WorkManager` service route on the ManagedCluster on port 443 (for gathering logs)
+- Inbound connectivity to the hub cluster's kube API server from the `ManagedCluster` on port 6443 (for Klusterlet/KlusterletAddon)
+- Inbound connectivity for post-commit hook from GitHub to the Hub
```
Comment (TheRealHaoLiu) [inline install/requirements.md]: optional for app feature related to GitHub @jnpacker can u help with the exact feature that need this

File: install/requirements.md [hunk 3]
```diff
@@ -24,3 +24,25 @@ You can access the Red Hat Advanced Cluster Management for Kubernetes console fr
...
 See the [Red Hat Advanced Cluster Management for Kubernetes Support matrix](https://access.redhat.com/articles/5058921) for additional information. 
+
+# Networking configuration
+
+Configure your network settings to allow the following connections:
+
+Hub cluster:
+- Outbound connectivity to cloud provider's API
+- Outbound connectivity to the Kubernetes API server of the Hive provisioned `ManagedCluster` on port 443 (so Hive-driven `openshift-install` is successful)
+- Outbound connectivity from the Hub to the channel source, including Github, Object Store, and Helm repository
```
Comment (TheRealHaoLiu) [inline install/requirements.md]: optional for app lifecycle feature using these sources

File: install/requirements.md [hunk 4]
```diff
@@ -24,3 +24,25 @@ You can access the Red Hat Advanced Cluster Management for Kubernetes console fr
...
+- Outbound and inbound connectivity to the `WorkManager` service route on the ManagedCluster on port 443 (for gathering logs)
+- Inbound connectivity to the hub cluster's kube API server from the `ManagedCluster` on port 6443 (for Klusterlet/KlusterletAddon)
+- Inbound connectivity for post-commit hook from GitHub to the Hub
+- Inbound connectivity from the Observatorium prometheus collector on the managed cluster to the Hub Thanos datastore
```
Comment (TheRealHaoLiu) [inline install/requirements.md]: not in 2.0

File: install/requirements.md [hunk 5]
```diff
@@ -24,3 +24,25 @@ You can access the Red Hat Advanced Cluster Management for Kubernetes console fr
...
+
+
+Managed cluster:
+- Inbound connectivity to the Kubernetes API server from the hub cluster on port 6443 (Hive)
+- Inbound connectivity to `WorkManager` service endpoint from the hub cluster on port 443 (for logging)
+- Outbound connectivity to the hub cluster's Kubernetes API server on port 6443 (for Klusterlet/KlusterletAddon)
+- Outbound connectivity from the Hub to channel source, including Github, Object Store, and Helm repository
+- Outbound connectivity from the Observatorium prometheus collector on the managed cluster to the Hub Thanos datastore
```
Comment (TheRealHaoLiu) [inline install/requirements.md]: not in 2.0

File: install/requirements.md (hunk 1 above)
Comment (TheRealHaoLiu) [inline install/requirements.md]: i think we just say "Outbound connectivity to the Kubernetes API server of the provisioned `ManagedCluster` on port 6443"

File: install/requirements.md [hunk 6]
```diff
@@ -24,3 +24,25 @@ You can access the Red Hat Advanced Cluster Management for Kubernetes console fr
...
+- Inbound connectivity to the hub cluster's kube API server from the `ManagedCluster` on port 6443 (for Klusterlet/KlusterletAddon)
+- Inbound connectivity for post-commit hook from GitHub to the Hub
+- Inbound connectivity from the Observatorium prometheus collector on the managed cluster to the Hub Thanos datastore
+
+
+Managed cluster:
+- Inbound connectivity to the Kubernetes API server from the hub cluster on port 6443 (Hive)
+- Inbound connectivity to `WorkManager` service endpoint from the hub cluster on port 443 (for logging)
+- Outbound connectivity to the hub cluster's Kubernetes API server on port 6443 (for Klusterlet/KlusterletAddon)
+- Outbound connectivity from the Hub to channel source, including Github, Object Store, and Helm repository
```
Comment (TheRealHaoLiu) [inline install/requirements.md]: optional

**Comment by cadawson** [comment]
Hi @TheRealHaoLiu . I updated this from your comments.  Does this apply to 1.0.1, as well, or should we just worry about 2.0? Thanks!

**Comment by cadawson** [comment]
Merging this, since review comments are in. If you have any additional issues, please open a new issue. Thanks!

**Pull request merged**